alacritty/src/display.rs

```diff
@@ -400,27 +400,33 @@ impl Display {
 
         let mut rects = lines.into_rects(&metrics, &size_info);
 
+        // Push visual bell after other terminal rects
```

Comment:
I think instead of `other terminal rects` we should specifically say `underline/strikeout rects` here. Just to make it clear that the message bar rect might still come in after it.